README.md

```diff
@@ -1,62 +1,86 @@
 
+- **Interoperability.** The ability to search information freely between other blockchains.
 
-## How to use
+To learn more from the KILT [Whitepaper](https://kilt.io/wp-content/uploads/2019/05/KILT-Whitepaper-v2019-May-28.pdf) or see our [how it works](https://kilt.io/kilt-data-sovereignty-and-interoperability/) video.
 
-Use within your project with `yarn add @kilt/sdk-js`
+## Documentation
 
-## Development setup
+KILT documentation is provided in several guides and demos.
 
-You can use different SDK branches or versions, by linking it into your projects locally.  
+- [Quick start guide](./docs/quick-start-guide.md)
+- [Getting started guide](./docs/getting-started.md)
+- [API documentation](https://kiltprotocol.github.io/sdk-js/api)
+- [KILT workshop](https://github.com/KILTprotocol/kilt-workshop-101)
+- [Demo client](https://kilt.io/developers-sub/kilt-demo-client/)
+- [Demo client code](https://github.com/KILTprotocol/demo-client)
 
-Execute `yarn link` in the SDK and copy the command in the output, which should look like this:
+To help improve, please see our [contribution page](./docs/contribution-guide.md).
 
-```yarn link "@kiltprotocol/sdk-js"```
+## How to install the SDK
 
-Go into your project folder and execute that second command.
+Install the KILT-SDK by running the following commands:
 
-The SDK is now symlinked in your projects `node_modules` folder
+```bash
 
-Before you see your changes from the SDK, you have to build it, by executing `yarn build`.
+    npm init -y
+    npm i @kiltprotocol/sdk-js
 
-### Removing the link
-Execute `yarn unlink "@kiltprotocol/sdk-js"` in the project folder.
+```
 
-After that execute `yarn install --check-files` to get the version from the registry back.
+Or with `yarn`:
 
-## Release / Deployment
+```bash
 
-Deployment is triggered by a push to the master branch as a result to a release build.
+    yarn init -y
+    yarn add @kiltprotocol/sdk-js
 
-To build a release, start the release build job for the SDK in *AWS CodeBuild*. See [here](https://github.com/KILTprotocol/release-build-job/blob/master/README.md#usage) for more info on building releases.
+```
 
-As a result of a release build, a new version of the SDK is published to the NPM registry.
+## Example
 
-*Note: Don't forget to reference the correct version in the client and services*
+More examples can be found within our [getting started guide](./docs/getting-started.md).
 
-## NB
+A claim type (CTYPE) can be credentials, of any kind, e.g. drivers license.
 
-Test coverage does not seem to be fail in all cases, except for testWatch.
+Building a claim must be done by the defined CTYPE respective fields.
 
-## FAQ
+Now we can easily create the KILT compliant claim. We have to include the full CType object, the raw claim object and the address of the owner/creator of the claim in the contstructor:
 
-### AWS build fails
+```TypeScript
 
-If the sdk build fails on AWS, please check the error log. Usually it says
+ const rawClaim = {
```

Comment:
Should we ident the code as code, to improve readability?

Reply:
Agreed, I thought I had done this. Thank you!